include/database.py

```diff
@@ -44,7 +44,9 @@ def insert_returning(connectionString, sql, parameters):
         cursor = connection.cursor()
 
         # Execute the query, note the rows
-        cursor.execute(sql, (parameters['Name'],))
+        #cursor.execute(sql, (parameters['Name'],))
+        for s_v in sql:
+            cursor.execute(s_v[0], s_v[1])
```

Comment:
I see what you are trying to do here, but you are actually complicating this code a lot more than it ha to be. Go back and look at what I wrote for the `select` method. Most of these operations should look very similar to that.